Backend/Models/Configurations/WorkerStateConfiguration.cs

```diff
@@ -6,15 +6,15 @@
 
 namespace Backend.Models.Configurations
 {
-    public class WorkerStateConfiguration : IEntityTypeConfiguration<WorkerState>
+    public class WorkerStateConfiguration : IEntityTypeConfiguration<EmployeeState>
```

Comment:
Configuration class name should also be refactored to reflect changes in model rename.

Reply:
Changed